app/controllers/api/current/appointments_controller.rb

```diff
@@ -18,6 +24,7 @@ def merge_if_valid(appointment_params)
       NewRelic::Agent.increment_metric('Merge/Appointment/schema_invalid')
       { errors_hash: validator.errors_hash }
     else
+      set_default_appointment_type(appointment_params)
       appointment = Appointment.merge(Api::Current::Transformer.from_request(appointment_params))
```

Comment:
this should happen only if the record is new

Reply:
Implemented change.